src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/build/targets/common.targets

```diff
@@ -147,9 +147,9 @@
       <ProjectsToPackage Include="@(CombinedNugetPkgProjects)" />      
     </ItemGroup>
     
-    <Message Text="Packaging..... @(ProjectsToPackage)" />
-    <Message Text="PackageOutputDir....... $(NugPkgOutPutDirPath)" />   
-    <Message Text="Starting Packaging..... @(ProjectsToPackage)" />
+    <Message Text="Packaging..... @(ProjectsToPackage)" Condition=" '$(ProjectsToPackage)' != '' " />
```

Comment:
This connection is not correct, it should be a `@` for what I think you want. However I think you handled this with the new condition on the Package target.

Reply:
thank you will fix it